perma_web/perma/models.py

```diff
@@ -944,3 +944,15 @@ def is_revisit(self):
         return self.__parsed.is_revisit()
 
 
+class UncaughtError(models.Model):
+    current_url = models.TextField()
+    user_agent = models.TextField()
+    stack = models.TextField()
+    name = models.TextField()
+    message = models.TextField()
+    custom_message = models.TextField()
+    user = models.ForeignKey(LinkUser, null=True, blank=True)
+    created_at = models.DateTimeField(default=timezone.now)
+
+    resolved = models.BooleanField(default=False)
+    resolved_by_user = models.CharField(null=True, max_length=767)
```

Comment:
This should be a `models.ForeignKey` rather than a CharField. (Then you'll need a related_name for the two ForeignKeys to distinguish them -- something like `related_name="errors_triggered"` for `user`, and `related_name="errors_resolved"` for `resolved_by_user`.)